wire-gradle-plugin/src/test/projects/configuration-cache-failure/gradle.properties

```diff
@@ -0,0 +1,4 @@
+org.gradle.caching=false
+org.gradle.unsafe.configuration-cache=true
+org.gradle.configuration-cache=true
+VERSION_NAME=5.0.0-SNAPSHOT
```

Comment:
omit this one? I don’t wanna have to keep it up-to-date

Reply:
It's required by the runner: https://github.com/square/wire/blob/b5bf223468acc788941fe088ddd098906a56aae2/wire-gradle-plugin/src/test/kotlin/com/squareup/wire/gradle/WirePluginTest.kt#L1492

Reply:
We have it somewhere else as well I think, our releasing steps update it so should be good